### DISCUSSION TITLE
Multi-password with access level

### DISCUSSION THREAD

**Discussion by elfux** [rokath/trice#460] [Ideas]
Different users would be able to see different levels of information.

For example:
`trice log -port COM13 -ts "ms" -password DevPassword`
access to every trice

`trice log -port COM13 -ts "ms" -password QAPassword`
access to generic information with some data, let's say that it's the same as "-pick info -pick error"

`trice log -port COM13 -ts "ms" -password ProductionPassword`
access to specific information, let's say that it's the same as "-pick msg"

The difference between this and "-pick" would be that the production couldn't change the flag and have access to the whole information.

**Comment by rokath** [comment]
This is a good idea, but implementing this as a part of trice will overburden the tool from my current point of view. At least there is then a configuration needed. How you will protect the config file? For your use case maybe some different batch files are sufficient. 

https://stackoverflow.com/questions/28174386/how-can-a-bat-file-be-converted-to-exe-without-third-party-tools?answertab=active#tab-top


https://unix.stackexchange.com/questions/64762/how-to-convert-a-shell-script-into-a-binary-executable
